_validate/validate.py

```diff
@@ -291,12 +315,17 @@ def main():
 		dest="file",
 		help="The json (.json) file containing add-on metadata."
 	)
+	parser.add_argument(
+		dest="versions",
```

Comment:
APIVersions I think would be a better name for this

```suggestion
		dest="APIVersions",
```